languagetool-language-modules/de/src/main/resources/org/languagetool/rules/de/grammar.xml

```diff
@@ -59540,6 +59540,64 @@ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301, USA
     <!-- Typografie -->
     <!-- ====================================================================== -->
     <category id="TYPOGRAPHY" name="Typografie">
+        <rulegroup id="SPACE_AFTER_BEFORE SLASH" name="Leerzeichen vor und nach Slash" default="temp_off"
+            <rule id="SPACE_AFTER_SLASH" name="Leerzeichen nach Slash">
```

Comment:
Siehe vorherigen Pull Request (PR): Subrules sollten keine ID kriegen.

Der "Umzug" und das Zusammenfassen der beiden Regel in eine Rulegroup sollte am besten im originalen PR stattfinden.